components/Tabs/Tabs.js

```diff
@@ -1,61 +1,63 @@
 class TabLink {
   constructor(tabElement){
     // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
+    this.tabElement = tabElement;
```

Comment:
I know all of this commenting and uncommenting is confusing, Overall you did a really nice job! Good work doing the stretch! Awesome!